data/filters/prepare-credit.lua

```diff
@@ -0,0 +1,87 @@
+-- Checks if any contributor information is available
+
+roles = {
+    "conceptualization",
+    "data-curation",
+    "formal-analysis",
+    "funding-acquisition",
+    "investigation",
+    "methodology",
+    "project-administration",
+    "resources",
+    "software",
+    "supervision",
+    "validation",
+    "visualization",
+    "writing-original-draft",
+    "writing-review-editing"
+}
+
+degrees = {
+    "Lead",
+    "Supporting",
+    "Equal"
+}
+
+function invalidRole(str)
```

Comment:
Suggestions: turn `roles` into a `pandoc.List` with `roles = pandoc.List{ ... }`. Then the role validity check becomes `roles:includes(str)`.  Same for `degrees`.

Reply:
Done in 976ffbf